packages/ui/components/pages/Fuse/FusePoolPage/index.tsx

```diff
@@ -112,14 +112,7 @@ const FusePoolPage = memo(() => {
                 />
                 <PoolStat
                   label="Utilization"
-                  value={
-                    data
-                      ? data.totalSuppliedFiat.toString() === '0'
-                        ? '0%'
-                        : ((data?.totalBorrowedFiat / data?.totalSuppliedFiat) * 100).toFixed(2) +
-                          '%'
-                      : undefined
-                  }
+                  value={data ? data.utilization.toFixed(2) + '%' : undefined}
```

Comment:
this pr does not seem to contain the code which is setting the `data.utilization` value. Was it already available all along?